package.json

```diff
@@ -24,13 +28,13 @@
   "homepage": "https://github.com/ciena-frost/frost-standard#readme",
   "devDependencies": {
     "chai": "^3.4.1",
-    "eslint": "^1.10.3",
+    "eslint": "^2.1.0",
     "istanbul": "^0.4.2",
     "mocha": "^2.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^4.1.7",
-    "eslint-config-standard": "^4.4.0",
+    "babel-eslint": "^5.0.0",
+    "eslint-config-standard": "^5.1.0",
```

Comment:
Looks like this may require another dependency now

https://travis-ci.org/ciena-frost/eslint-config-frost-standard/builds/109890304